integration_tests/testsuite/deploy_app.py

```diff
@@ -43,15 +47,18 @@ def deploy_app(image, appdir):
             fout.close()
         fin.close()
 
+        global DEPLOYED_VERSION
+        DEPLOYED_VERSION = test_util.generate_version()
```

Comment:
Could you make this a parameter to the cleanup function instead of storing it as a global?